app/assets/stylesheets/base.css.scss

```diff
@@ -81,9 +81,17 @@ a {
     margin: 0 auto;
 }
 
+.profpiccircle {
+  border-radius:50%;
+}
+
+.row {
+  margin: 15px;
+}
+
 .box {
```

Comment:
Note that changing `.box`, `.row`, `.h1` and `.h2` have far-reaching consequences. Have you checked that all the other pages using these styles still look right / responsive?